### PR TITLE
Bug 1205220 - SyncManager - Sync app glue. r=michielbdjong

### PR DESCRIPTION
**Pull request by ferjm**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2becb1faa439f1b24579dfe82357791ccbfddb28

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bd451cd0159bf18b01e8af9c126a944f3c0d77ce

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bd1c3475a6dc48a8172268c86532d51b6481be88

File: apps/sync/index.html [hunk 1]
```diff
@@ -5,15 +5,12 @@
     <meta charset="utf-8">
     <!-- Lazy loaded shared
     <script defer src="shared/js/async_storage.js"></script>
+    <script defer src="shared/js/sync/errors.js"></script>
```
Comment (michielbdejong) [inline apps/sync/index.html]: Should we list all LazyLoaded files here?

File: apps/sync/test/unit/bootstrap_test.js [hunk 2]
```diff
@@ -11,36 +11,65 @@
...
-  suite('when script is loaded', () => {
+
+  mocksForBootstrap.attachTestHelpers();
+
+  suite('when script is loaded', function() {
+    var addEventListenerSpy;
+
+    suiteSetup(function(done) {
+      addEventListenerSpy = sinon.spy(window, 'addEventListener');
+      requireApp('sync/js/bootstrap.js', done);
```
Comment (michielbdejong) [inline apps/sync/test/unit/bootstrap_test.js]: Maybe add a comment that the other suites rely on this suite running first.

File: apps/sync/index.html (hunk 1 above)
Comment (ferjm) [inline apps/sync/index.html]: Only the ones that come from shared so the build process can add them to the apps folder.

Comment (michielbdejong) [inline apps/sync/index.html]: Ah right, good to know!

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5c16376098735e1b4246aa27232355db7d5dfad8

**Pull request merged**